packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2410,7 +2410,9 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       modelColumn,
       addToExisting
     );
-    this.updateSorts(sorts);
+    if (model.columns[columnIndex].isSortable) {
+      this.updateSorts(sorts);
+    }
```

Comment:
We should add a `log.debug` in an `else` statement here indicating we didn't sort because the column wasn't sortable.